api/views/event_view.py

```diff
@@ -13,18 +13,52 @@
 )
 
 from .team_base import *
-from ..permissions import IsUserOwnerOrReadOnly
 
 class EventViewSet(TeamRelatedViewSet):
     queryset = Event.objects.all()
     serializer_class = EventSerializer
     permission_classes = [
         permissions.IsAuthenticated,
-        IsUserOwnerOrReadOnly | IsAdminMemberOrReadOnly,
+        IsAdminMemberOrReadOnly,
         IsMemberUser,
     ]
     filter_backends = [django_filters.rest_framework.DjangoFilterBackend]
     search_fields = ['^team__name', '^title', '^description', '^description', '^event_type',
                      '^user']
     ordering_fields = ['created', 'team', 'event_type', 'user']
-    
\ No newline at end of file
+
+    @action(methods=["GET"],
+            permission_classes=[permissions.IsAuthenticated, IsMemberUser]
+    )
+    def list(self, request, pk=None):
+        serializer = self.serializer_class(self.queryset, many=True)
```

Comment:
listing from a pk doesn't really make sense, as this effectively only retrieves a single object. Remember, you're inheriting `list_team` from TeamRelatedBase, which will return a queryset with all the objects (in this case events) whose team foreign key matches up with the pk parameter.